.github/workflows/on-released.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   publish:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
I would go either to 24.04 or just to the latest.
Let's delay the next deprecation :)

Reply:
Good point, for some reason I thought that 22.04 was the latest...
Done :)